zio-http/src/main/scala/zio/http/Body.scala

```diff
@@ -125,10 +125,12 @@ trait Body { self =>
    */
   def isEmpty: Boolean
 
+  final def getMediaType: Option[MediaType] = this.mediaType
```

Comment:
ZIO HTTP does not use getter-style conventions, except for `is` / `has` for boolean-returning methods. So this should be `mediaType`. Note there is already a `mediaType` that is private and serves the same function as this, so we should just make that one public.

Reply:
In addition, we would have a 'setter' which is `def mediaType(newMediaType: MediaType): Body`.